conf/test_full.config

```diff
@@ -15,10 +15,11 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     // Input data for full size test
-    input = "${projectDir}/tests/csv/3.0/test_full_data.csv"
+    input = '${projectDir}/tests/csv/3.0/test_full_data.csv'
 
-    wes   = true
-
-    // Ignore `--input` as otherwise the parameter validation will throw an error
+    // Other params
+    tools = 'strelka,freebayes,haplotypecaller,vep'
```

Comment:
How about one or snp callers, one sv one, and one cna caller  + annotation? 

strelka, manta, ascat, vep?

Reply:
If we wanna link the full size tests to benchmarking, then we should run all variant callers that allow germline calling. I can also add a structural caller and CNA caller